fe/fe-core/src/main/java/org/apache/doris/qe/SessionVariable.java

```diff
@@ -927,6 +942,10 @@ public void setEnableNereidsTrace(boolean enableNereidsTrace) {
         this.enableNereidsTrace = enableNereidsTrace;
     }
 
+    public void setEnableNereidsTimeout(boolean enableNereidsTimeout) {
+        this.enableNereidsTimeout = enableNereidsTimeout;
+    }
+
```

Comment:
Because variable is public, we don't need get/set

Reply:
done